packages/manager/modules/pci/src/components/project/images-list/images-list.service.js

```diff
@@ -3,12 +3,71 @@ import groupBy from 'lodash/groupBy';
 import map from 'lodash/map';
 import pick from 'lodash/pick';
 import omit from 'lodash/omit';
+import snakeCase from 'lodash/snakeCase';
 
 import Image from './images.class';
 
+function getDistribution(name, type) {
+  const regex = {
+    linux: [
```

Comment:
Not sure it is a good idea to map distributions this way. Each time a new distribution is added or updated we might have to update and maintain regexps

Reply:
It was made like this into OVH-API-SERVICES.
I just import the function like it.

@marie-j do you have a suggestion ?

Reply:
@mohammed-zahaf Just because it was done that way doesn't mean it should be repeated 😄 
What is the need here ?

Reply:
No, I let it to preserve the behavior.
Currently, from returned images list we don't have the `distribution` attribute.

So this function was made to deduce this attribute from the `name` attribute.
But sure it was to be provided via an attribute.

Reply:
Sample:
`
creationDate: "2020-02-06T15:32:47Z"
flavorType: null
id: "blblablabla"
minDisk: 0
minRam: 0
name: "Debian 9 - IaaS hardened"
planCode: null
region: "GRA5"
size: 0.70184326171875
status: "active"
tags: []
type: "linux"
user: "root"
visibility: "private"
`

Reply:
So @marie-j as i think, the API don't give us this info.
I asked the PCI team they said it will be planned in the future.

So we can let it like this until we have this attribute.